source/features/index.tsx

```diff
@@ -103,6 +105,8 @@ const globalReady: Promise<RGHOptions> = new Promise(async resolve => {
 		getLocalStrings(),
 	]);
 
+	console.log({value: _`Original`});
```

Comment:
Line 98 and line 108 show the regular string and then the string after the replacement

<img width="271" alt="Screen Shot 1" src="https://user-images.githubusercontent.com/1402241/176991389-cfe86c07-162b-44e4-bbf4-efb7c66ee77e.png">